demisto_sdk/commands/common/logger.py

```diff
@@ -25,18 +26,42 @@
 DATE_FORMAT = "%Y-%m-%dT%H:%M:%S"
 
 DEPRECATED_PARAMETERS = {
```

Comment:
were these underscored before deprecation?

Reply:
i'm just changing the new name, which use underscore and not hyphens